ui/src/server/auth.ts

```diff
@@ -38,16 +40,27 @@ declare module "next-auth" {
  */
 export const authOptions: NextAuthOptions = {
   callbacks: {
-    session: ({ session, user }) => ({
+    session: ({ session, token }) => ({
       ...session,
       user: {
         ...session.user,
-        id: user.id,
+        id: token.sub,
       },
     }),
   },
-  adapter: DrizzleAdapter(db, mysqlTable) as Adapter,
+  session: {
+    strategy: "jwt",
+  },
+  adapter: sqlDrizzleAdapter(db as unknown as NeonDatabase) as Adapter,
   providers: [
+    DiscordProvider({
+      clientId: process.env.DISCORD_CLIENT_ID ?? "",
+      clientSecret: process.env.DISCORD_CLIENT_SECRET ?? "",
+    }),
+    GitHubProvider({
+      clientId: process.env.GITHUB_ID ?? "",
+      clientSecret: process.env.GITHUB_SECRET ?? "",
```

Comment:
Env vars in t3 are done cia t3 env, checkout the env.mjs file

Reply:
Added there as well